clang/include/clang/Driver/Types.def

```diff
@@ -125,12 +125,7 @@ TYPE("tempfiletable",            Tempfiletable,INVALID,         "table",  phases
 TYPE("tempAOCOfilelist",         TempAOCOfilelist, INVALID,     "txt",    phases::Compile, phases::Backend, phases::Assemble, phases::Link)
 TYPE("archive",                  Archive,      INVALID,         "a",      phases::Compile, phases::Backend, phases::Assemble, phases::Link)
 TYPE("wholearchive",             WholeArchive, INVALID,         "a",      phases::Compile, phases::Backend, phases::Assemble, phases::Link)
-TYPE("fpga_aocx",                FPGA_AOCX,    INVALID,         "aocx",   phases::Compile, phases::Backend, phases::Assemble, phases::Link)
-TYPE("fpga_aocr",                FPGA_AOCR,    INVALID,         "aocr",   phases::Compile, phases::Backend, phases::Assemble, phases::Link)
-TYPE("fpga_aocr_emu",            FPGA_AOCR_EMU, INVALID,        "aocr",   phases::Compile, phases::Backend, phases::Assemble, phases::Link)
-TYPE("fpga_aoco",                FPGA_AOCO,    INVALID,         "aoco",   phases::Compile, phases::Backend, phases::Assemble, phases::Link)
 TYPE("fpga_dep",                 FPGA_Dependencies, INVALID,    "d",      phases::Compile, phases::Backend, phases::Assemble, phases::Link)
```

Comment:
Should this not be removed?

Reply:
Not yet - there are still usages of this.  Ultimately, yes this will be removed but probably in the next phase of the cleanup.

Reply:
I noticed that you removed `FPGA_Dependencies_List`. Since the usages of `FPGA_Dependencies` are mostly located near `FPGA_Dependencies_List`, any reason why it was left out?

Reply:
The usage of `FPGA_Dependencies` is not specifically tied to the FPGA archive support, and this round of cleanup is specific to the FPGA archives.